apps/src/lib/client/rpc.rs

```diff
@@ -821,16 +1069,65 @@ pub async fn get_public_key(
 /// Check if the given address is a known validator.
 pub async fn is_validator(
     address: &Address,
+    epoch: Option<Epoch>,
```

Comment:
I don't think we should add the epoch arg here, as the current implementation and usage is to check if a given address is a validator, but it doesn't care about whether it's in the active or inactive validator set (the active set size is limited to 128 slots, but this is done only for the consensus for Tendermint). I think even the validators that are in the inactive set can participate in governance.